testing/asserts.ts

```diff
@@ -539,10 +536,22 @@ export function assertObjectMatch(
 
     function fn(a: loose, b: loose): loose {
       // Prevent infinite loop with circular references with same filter
-      if ((seen.has(a)) && (seen.get(a) === b)) {
+      if (seen.has(a) && seen.get(a) === b) {
         return a;
       }
-      seen.set(a, b);
+
+      try {
```

Comment:
Real change is catching `WeakMap.set()`, which will throw a TypeError for primitives and non-iterables. The other changes are `deno fmt`